.github/workflows/release.yml

```diff
@@ -1,22 +0,0 @@
-name: Create release
-
-on:
-  push:
-    tags:
-      - '*'
```

Comment:
This apparently never ran, I think it should have been `- *` but since it never ran, we can probably remove it.